src/components/SeedInfo/SeedInfoViews/FungalShifts.tsx

```diff
@@ -1,83 +1,110 @@
-import React, { useContext } from 'react';
-import classnames from 'classnames';
+import { useContext } from 'react';
 import { Stack, Form, Tooltip, OverlayTrigger } from 'react-bootstrap';
 
 import GameInfoProvider from '../../../services/SeedInfo/infoHandler';
 import { capitalize } from '../../../services/helpers';
 import { FungalInfoProvider } from '../../../services/SeedInfo/infoHandler/InfoProviders/Fungal';
 import { AlchemyConfigContext } from '../../AlchemyConfigContext';
 import { useTranslation } from 'react-i18next';
-import classNames from 'classnames';
 import { useMaterialFavorite } from './helpers';
+import { MaterialInfoProvider } from '../../../services/SeedInfo/infoHandler/InfoProviders/Material';
 
 const Flask = () => {
 	return <b className="text-info">Flask</b>;
 };
 
+interface IFungalMaterialProps {
+	materials: Map<string, string>
+	direction: boolean
+	isFlask: boolean
+	isFavorite // TODO: add type 
+	showId // TODO: add type 
+}
+
+const FungalMaterial = (props: IFungalMaterialProps) => {
```

Comment:
```suggestion
const FungalMaterial: React.FC<IFungalMaterialProps> = ({...distruct props}) => {
```

Reply:
So they are like regular vars and not `props.*`

Reply:
When I add that line, I get

```
(parameter) props: any
',' expected.ts(1005)
Property 'props' does not exist on type 'IFungalMaterialProps'.ts(2339)
```

I don't know enough to fix this. Initially, I just tried to do it as in the other functions.

```
const FungalMaterial = (props: IFungalMaterialProps) => { // my addition
const Shift = (props: IShiftProps) => { // existing
const FungalShifts = (props: IFungalShiftsProps) => { // existing
```

What should I do?

Reply:
Done.